generators/app/templates/config/locales/pt-BR/index.js

```diff
@@ -1,9 +0,0 @@
-import base from './base.yml';
-import models from './models.yml';
-
-let pt_BR = {
-  ...base['pt-BR'],
-  ...models['pt-BR']
-};
-
-export { pt_BR };
```

Comment:
This was all made with ReactIntl and Rails in mind, so that we'd have a simple way to reuse the exactly same yaml file for both, client and server, without the need for duplication.

The only bad thing was for now having to configure these js files by hand, that's why I would add two subgenerators for creating new locales (like pt-BR) and new locale files (like models.yml) soon enough. The idea was to have these subgenerators take care of all that stuff.